### PR TITLE
Colorconverter cleanup

### PR DESCRIPTION
**Pull request by machour**
* Remove unnecessary parentheses
* Use single quotes
* Use short syntax
* Strict comparison for strings
* Clean up ifs
* Add missing phpdoc annotations

### CODE REVIEW

File: src/Color/ColorConverter.php [hunk 1]
```diff
@@ -199,8 +213,8 @@ private function processHashColor($color)
 	/**
 	 * @param $mode
 	 * @param mixed[] $cores
-	 *
-	 * @return bool|\float[]
+	 * @return bool|float[]
+	 * @throws \Mpdf\MpdfException
```
Comment (finwe) [inline src/Color/ColorConverter.php]: I discourage `@throws` PHPDoc tag, it is a mess to maintain, better to ommit it.

Comment (Klap-in) [reply src/Color/ColorConverter.php]: Could it be useful if automatically docs are generated from the source?

Comment (finwe) [reply src/Color/ColorConverter.php]: IMHO not too much. There is not enough difference in exceptions thrown (now there are about 2 or 3 types) and when the exception change in code is not reflected in the phpdoc, the generated doc would be off as well.

Comment (machour) [reply src/Color/ColorConverter.php]: I agree it's a pity to maintain, reverted the change :)

**Pull request merged**